src/machines.scss

```diff
@@ -101,3 +101,11 @@
     background-color: var(--pf-v5-global--BackgroundColor--dark-100);
   }
 }
+
+// radio buttons rendered vertically in pf-m-horizontal https://github.com/patternfly/patternfly/issues/5787
+.pf-v5-c-form__group-control.pf-m-inline {
+  .pf-v5-c-radio {
+    width: auto;
+    display: grid;
+  }
```

Comment:
Hmm, is that obsolete with https://github.com/cockpit-project/cockpit/commit/27832047ec3b058b4e2ace316e117f15769d47d9 now? This was supposed to fix it everywhere.